src/tools/dev/scripts/bv_support/bv_adios2.sh

```diff
@@ -235,6 +211,15 @@ function build_adios2
         cfg_opts="${cfg_opts} -DCMAKE_CXX_FLAGS:STRING=\"${CXX_OPT_FLAGS}\""
         cfg_opts="${cfg_opts} -DADIOS2_USE_SST:BOOL=ON"
 
+        # Use Blosc?
+        if [[ "$DO_BLOSC" == "yes" ]] ; then
+            BLOSC_INCLUDE_DIR="${VISITDIR}/blosc/${BLOSC_VERSION}/${VISITARCH}/include"
```

Comment:
Items like BLOSC_INCLUDE_DIR, BLOSC_LIBRARY should be exported by bv_blosc.sh so other modules can use them directly, rather then each module needing to reconstruct from scratch.